.babelrc

```diff
@@ -1,14 +1,18 @@
 {
   "presets": [
-    ["env", {
-      "targets": {
-        "browsers": ["last 2 versions", "iOS >=10", "ie >= 10"]
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [ "last 2 versions", "iOS >= 10", "ie >= 10" ]
```

Comment:
Not sure on this list, but this might need to align with the terra-supported browserlist. Likely an issue for another PR. https://github.com/cerner/browserslist-config-terra